test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -577,6 +582,87 @@ def __set_test_end_data(self, test, err=None):
         test.duration = test.end_time - test.start_time
         test.captured_logs = self.result.handler.get_logs()
 
+    def __get_param_dictionary(self):
+        """
+        Method to convert data in param into dictionary of counts
+        """
+        paramkeys = ['server', 'servers', 'mom', 'moms', 'comms', 'client',
```

Comment:
"comm" is not an option since currently PTL has only below param input types for PBS daemon hostnames and there is no "comm" specification here, but only "comms" specification being present:

    :param server: The hostname on which the PBS ``server/scheduler``
                   are running
    :param mom: The hostname on which the PBS MoM is running
    :param servers: Colon-separated list of hostnames hosting a PBS server.
                    Servers are then accessible as a dictionary in the
                    instance variable servers.
    :param client: For CLI mode only, name of the host on which the PBS
                   client commands are to be run from. Format is
                   ``<host>@<path-to-config-file>``
    :param moms: Colon-separated list of hostnames hosting a PBS MoM.
                 MoMs are made accessible as a dictionary in the instance
                 variable moms.
    :param comms: Colon-separated list of hostnames hosting a PBS Comm.
                  Comms are made accessible as a dictionary in the
                  instance variable comms.
    :param nomom=<host1>\:<host2>...: expect no MoM on given set of hosts